packages/adblocker-benchmarks/run.js

```diff
@@ -87,6 +87,11 @@ function loadLists() {
   const filename = HOSTS_ONLY ? 'hosts.txt' : 'easylist.txt';
   let content = fs.readFileSync(path.resolve(__dirname, filename), { encoding: 'utf-8' });
 
+  if (!HOSTS_ONLY) {
+    content += fs.readFileSync(path.resolve(__dirname, 'easyprivacy.txt'), { encoding: 'utf-8' })
+               .replace(/^\[Adblock\b.*\n/, '');
```

Comment:
Since we're concatenating lists we need to remove the "[Adblock Plus x.x]" at the top, lest it be interpreted as a filter.